examples/angular-example/package.json

```diff
@@ -21,11 +21,15 @@
     "@angular/router": "^18.0.0",
     "@uppy/angular": "workspace:*",
     "@uppy/core": "workspace:*",
+    "@uppy/dashboard": "workspace:*",
     "@uppy/drag-drop": "workspace:*",
     "@uppy/google-drive": "workspace:*",
     "@uppy/progress-bar": "workspace:*",
+    "@uppy/status-bar": "workspace:*",
```

Comment:
we have an issue currently were using the framework packages requires you to install all dependencies, even if you don't use them. We need to fix this, likely with export maps, but it's a breaking change. We can add them for now but ideally we remove unused deps again once we fixed this